lib/improver/tests/blending/weights/test_ChooseWeightsLinear.py

```diff
@@ -557,20 +550,83 @@ def test_spatial_varying_weights(self):
                                weights_cube.metadata)
 
 
+class Test__slice_input_cubes(IrisTest):
```

Comment:
Seeing as this only has 2 unit tests and there are 3 paths through this function we're missing a unit test. I think it's the  `if len(cubes.coord(self.config_coord_name).points) == 1:` branch you aren't testing,